run

```diff
@@ -15,4 +15,14 @@ popd  > /dev/null
 
 TARGET="${TARGET:-release}"
 
-exec "$SCRIPT_PATH/target/$TARGET/compiler-cli" "$@"
+if [[ $BENCH == "1" ]]; then
+    VANILLA=1 exec "$SCRIPT_PATH/target/$TARGET/compiler-cli" "$@"
+else
+    if [[ $# == 1 ]] ; then
+        VANILLA=1 exec "$SCRIPT_PATH/target/$TARGET/compiler-cli" "--compile" "-O" "moderate" "$1"
+    elif [[ $# == 2 ]] && [[ $1 == "--compile" ]]; then
+        VANILLA=1 exec "$SCRIPT_PATH/target/$TARGET/compiler-cli" "--compile" "-O" "moderate" "$2"
+    elif [[ $# == 2 ]] && [[ $1 == "--compile-firm" ]]; then
+        VANILLA=1 exec "$SCRIPT_PATH/target/$TARGET/compiler-cli" "--compile-firm" "-O" "moderate" "$2"
+    fi
+fi
```

Comment:
Missing error handling: exit with nonzero status code 123 and a clear indication that the run script was invoked with unexpected args